sdk/communication/azure-communication-chat/tests/test_chat_thread_client_e2e.py

```diff
@@ -149,12 +153,27 @@ def test_list_participants(self):
         if self.is_live:
             time.sleep(2)
```

Comment:
if you need to do this kind of stuff, i.e. `time.sleep()` in the test code, due to service behavior, let's not write live tests for these scenarios, this causes inconsistency/unreliability issues in our live tests. Instead mocking the network call - as you already added. 
We are removing these kinds of tests from the e2e tests in `master` branch too.